sp19/syllabus.yml

```diff
@@ -46,9 +46,15 @@ primary:
 - name: ""
   file: "recurrent-nets"
   covr: ""
-  tags: ["neural-nets", "recurrent-nets", ]
-  inst: ["ionlights"]
+  tags: ["neural-nets", "recurrent-nets", "LSTMs", "Embeddings"]
+  inst: ["ionlights", "brandons209"]
```

Comment:
@brandons209 prepend your name, i'm essentially an editor on this, even though i'm anticipating major usage of my source material. 😅